content/id/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Mengatur Master Kubernetes dengan Ketersediaan Tinggi (High-Availability)
```

Comment:
let's use Control Plane for this instead of master, this is kind of banned word in tech community now
```suggestion
title: Mengatur Control Plane Kubernetes dengan Ketersediaan Tinggi (High-Availability)
```

Reply:
Agree!